### PR TITLE
Update aquaFargate.yaml

### PR DESCRIPTION
**Pull request by cpt-redbeard**
Adding gRPC

### CODE REVIEW

**Comment by rshmiel** [comment]
@cpt-redbeard  can you please introduce yourself, are you an aqua user? if so please contact me in teams.

File: cloudformation/aqua-ecs-fargate/aquaFargate.yaml [hunk 1]
```diff
@@ -196,8 +196,8 @@ Resources:
           - aqua-console
       RequiresCompatibilities:
         - FARGATE
-      Cpu: '512'
-      Memory: '1024'
+      Cpu: '406'
```
Comment (rkwa-dan) [inline cloudformation/aqua-ecs-fargate/aquaFargate.yaml:199]: did you mean 4096 CPU ?

Comment (cpt-redbeard) [reply cloudformation/aqua-ecs-fargate/aquaFargate.yaml:199]: Yes

**Comment by semyonmor** [comment]
HI @cpt-redbeard 
The memory has the required definition.
https://github.com/aquasecurity/aqua-aws/blob/f697ea31241dfa5e2b6b12ba3385499a31984f8e/cloudformation/aqua-ecs-fargate/aquaFargate.yaml#L265
Closing the PR

**Pull request closed without merging**